# Logfile created on 2016-11-19 13:02:12 by logger.rb/47272
I, [2016-11-19T13:02:13.206754 #11969]  INFO -- : *** below you find the most recent exception thrown, this will be likely (but not certainly) the exception that made the application exit abnormally ***
E, [2016-11-19T13:02:13.206892 #11969] ERROR -- : Value will be available once the pipeline executes. (Redis::FutureNotReady)

I, [2016-11-19T13:02:13.206929 #11969]  INFO -- : *** below you find all exception objects found in memory, some of them may have been thrown in your application, others may just be in memory because they are standard exceptions ***
E, [2016-11-19T13:02:13.242937 #11969] ERROR -- : no such frame (ArgumentError)
/root/.rbenv/versions/2.2.1/lib/ruby/gems/2.2.0/gems/binding_of_caller-0.7.2/lib/binding_of_caller/mri2.rb:25:in `frame_binding'
/root/.rbenv/versions/2.2.1/lib/ruby/gems/2.2.0/gems/binding_of_caller-0.7.2/lib/binding_of_caller/mri2.rb:25:in `block (2 levels) in callers'
/root/.rbenv/versions/2.2.1/lib/ruby/gems/2.2.0/gems/binding_of_caller-0.7.2/lib/binding_of_caller/mri2.rb:23:in `loop'
/root/.rbenv/versions/2.2.1/lib/ruby/gems/2.2.0/gems/binding_of_caller-0.7.2/lib/binding_of_caller/mri2.rb:23:in `block in callers'
/root/.rbenv/versions/2.2.1/lib/ruby/gems/2.2.0/gems/binding_of_caller-0.7.2/lib/binding_of_caller/mri2.rb:21:in `open'
/root/.rbenv/versions/2.2.1/lib/ruby/gems/2.2.0/gems/binding_of_caller-0.7.2/lib/binding_of_caller/mri2.rb:21:in `callers'
/root/.rbenv/versions/2.2.1/lib/ruby/gems/2.2.0/gems/better_errors-1.1.0/lib/better_errors/core_ext/exception.rb:9:in `block in <class:Exception>'
/root/.rbenv/versions/2.2.1/lib/ruby/2.2.0/open-uri.rb:36:in `initialize'
/root/.rbenv/versions/2.2.1/lib/ruby/2.2.0/open-uri.rb:36:in `open'
/root/.rbenv/versions/2.2.1/lib/ruby/2.2.0/open-uri.rb:36:in `open'
/root/.rbenv/versions/2.2.1/lib/ruby/2.2.0/logger.rb:628:in `open_logfile'
/root/.rbenv/versions/2.2.1/lib/ruby/2.2.0/logger.rb:584:in `initialize'
/root/.rbenv/versions/2.2.1/lib/ruby/2.2.0/logger.rb:318:in `new'
/root/.rbenv/versions/2.2.1/lib/ruby/2.2.0/logger.rb:318:in `initialize'
/root/.rbenv/versions/2.2.1/lib/ruby/gems/2.2.0/gems/daemons-1.1.9/lib/daemons/application.rb:353:in `new'
/root/.rbenv/versions/2.2.1/lib/ruby/gems/2.2.0/gems/daemons-1.1.9/lib/daemons/application.rb:353:in `exception_log'
/root/.rbenv/versions/2.2.1/lib/ruby/gems/2.2.0/gems/daemons-1.1.9/lib/daemons/application.rb:165:in `block in start_load'
E, [2016-11-19T13:02:13.259073 #11969] ERROR -- : No such file or directory @ rb_sysopen - /root/coinroots/log/peatio:amqp:deposit_coin.log (Errno::ENOENT)
/root/.rbenv/versions/2.2.1/lib/ruby/2.2.0/open-uri.rb:36:in `initialize'
/root/.rbenv/versions/2.2.1/lib/ruby/2.2.0/open-uri.rb:36:in `open'
/root/.rbenv/versions/2.2.1/lib/ruby/2.2.0/open-uri.rb:36:in `open'
/root/.rbenv/versions/2.2.1/lib/ruby/2.2.0/logger.rb:628:in `open_logfile'
/root/.rbenv/versions/2.2.1/lib/ruby/2.2.0/logger.rb:584:in `initialize'
/root/.rbenv/versions/2.2.1/lib/ruby/2.2.0/logger.rb:318:in `new'
/root/.rbenv/versions/2.2.1/lib/ruby/2.2.0/logger.rb:318:in `initialize'
/root/.rbenv/versions/2.2.1/lib/ruby/gems/2.2.0/gems/daemons-1.1.9/lib/daemons/application.rb:353:in `new'
/root/.rbenv/versions/2.2.1/lib/ruby/gems/2.2.0/gems/daemons-1.1.9/lib/daemons/application.rb:353:in `exception_log'
/root/.rbenv/versions/2.2.1/lib/ruby/gems/2.2.0/gems/daemons-1.1.9/lib/daemons/application.rb:165:in `block in start_load'
E, [2016-11-19T13:02:13.259424 #11969] ERROR -- : no such frame (ArgumentError)
/root/.rbenv/versions/2.2.1/lib/ruby/gems/2.2.0/gems/binding_of_caller-0.7.2/lib/binding_of_caller/mri2.rb:25:in `frame_binding'
/root/.rbenv/versions/2.2.1/lib/ruby/gems/2.2.0/gems/binding_of_caller-0.7.2/lib/binding_of_caller/mri2.rb:25:in `block (2 levels) in callers'
/root/.rbenv/versions/2.2.1/lib/ruby/gems/2.2.0/gems/binding_of_caller-0.7.2/lib/binding_of_caller/mri2.rb:23:in `loop'
/root/.rbenv/versions/2.2.1/lib/ruby/gems/2.2.0/gems/binding_of_caller-0.7.2/lib/binding_of_caller/mri2.rb:23:in `block in callers'
/root/.rbenv/versions/2.2.1/lib/ruby/gems/2.2.0/gems/binding_of_caller-0.7.2/lib/binding_of_caller/mri2.rb:21:in `open'
/root/.rbenv/versions/2.2.1/lib/ruby/gems/2.2.0/gems/binding_of_caller-0.7.2/lib/binding_of_caller/mri2.rb:21:in `callers'
/root/.rbenv/versions/2.2.1/lib/ruby/gems/2.2.0/gems/better_errors-1.1.0/lib/better_errors/core_ext/exception.rb:9:in `block in <class:Exception>'
/root/.rbenv/versions/2.2.1/lib/ruby/gems/2.2.0/gems/bunny-1.2.1/lib/bunny/consumer_work_pool.rb:55:in `join'
/root/.rbenv/versions/2.2.1/lib/ruby/gems/2.2.0/gems/bunny-1.2.1/lib/bunny/consumer_work_pool.rb:55:in `block in join'
/root/.rbenv/versions/2.2.1/lib/ruby/gems/2.2.0/gems/bunny-1.2.1/lib/bunny/consumer_work_pool.rb:55:in `each'
/root/.rbenv/versions/2.2.1/lib/ruby/gems/2.2.0/gems/bunny-1.2.1/lib/bunny/consumer_work_pool.rb:55:in `join'
/root/coinroots/lib/daemons/amqp_daemon.rb:81:in `<top (required)>'
/root/.rbenv/versions/2.2.1/lib/ruby/gems/2.2.0/gems/daemons-1.1.9/lib/daemons/application.rb:203:in `load'
/root/.rbenv/versions/2.2.1/lib/ruby/gems/2.2.0/gems/daemons-1.1.9/lib/daemons/application.rb:203:in `start_load'
/root/.rbenv/versions/2.2.1/lib/ruby/gems/2.2.0/gems/daemons-1.1.9/lib/daemons/application.rb:298:in `start'
/root/.rbenv/versions/2.2.1/lib/ruby/gems/2.2.0/gems/daemons-1.1.9/lib/daemons/controller.rb:70:in `run'
/root/.rbenv/versions/2.2.1/lib/ruby/gems/2.2.0/gems/daemons-1.1.9/lib/daemons.rb:147:in `block in run'
/root/.rbenv/versions/2.2.1/lib/ruby/gems/2.2.0/gems/daemons-1.1.9/lib/daemons/cmdline.rb:109:in `call'
/root/.rbenv/versions/2.2.1/lib/ruby/gems/2.2.0/gems/daemons-1.1.9/lib/daemons/cmdline.rb:109:in `catch_exceptions'
/root/.rbenv/versions/2.2.1/lib/ruby/gems/2.2.0/gems/daemons-1.1.9/lib/daemons.rb:146:in `run'
/root/.rbenv/versions/2.2.1/lib/ruby/gems/2.2.0/gems/daemons-rails-1.2.1/lib/daemons/rails.rb:16:in `run'
lib/daemons/deposit_coin_ctl:11:in `<main>'
E, [2016-11-19T13:02:13.269193 #11969] ERROR -- : no such frame (ArgumentError)
/root/.rbenv/versions/2.2.1/lib/ruby/gems/2.2.0/gems/binding_of_caller-0.7.2/lib/binding_of_caller/mri2.rb:25:in `frame_binding'
/root/.rbenv/versions/2.2.1/lib/ruby/gems/2.2.0/gems/binding_of_caller-0.7.2/lib/binding_of_caller/mri2.rb:25:in `block (2 levels) in callers'
/root/.rbenv/versions/2.2.1/lib/ruby/gems/2.2.0/gems/binding_of_caller-0.7.2/lib/binding_of_caller/mri2.rb:23:in `loop'
/root/.rbenv/versions/2.2.1/lib/ruby/gems/2.2.0/gems/binding_of_caller-0.7.2/lib/binding_of_caller/mri2.rb:23:in `block in callers'
/root/.rbenv/versions/2.2.1/lib/ruby/gems/2.2.0/gems/binding_of_caller-0.7.2/lib/binding_of_caller/mri2.rb:21:in `open'
/root/.rbenv/versions/2.2.1/lib/ruby/gems/2.2.0/gems/binding_of_caller-0.7.2/lib/binding_of_caller/mri2.rb:21:in `callers'
/root/.rbenv/versions/2.2.1/lib/ruby/gems/2.2.0/gems/better_errors-1.1.0/lib/better_errors/core_ext/exception.rb:9:in `block in <class:Exception>'
/root/.rbenv/versions/2.2.1/lib/ruby/gems/2.2.0/gems/bunny-1.2.1/lib/bunny/reader_loop.rb:94:in `raise'
/root/.rbenv/versions/2.2.1/lib/ruby/gems/2.2.0/gems/bunny-1.2.1/lib/bunny/reader_loop.rb:94:in `raise'
/root/.rbenv/versions/2.2.1/lib/ruby/gems/2.2.0/gems/bunny-1.2.1/lib/bunny/session.rb:796:in `maybe_shutdown_reader_loop'
/root/.rbenv/versions/2.2.1/lib/ruby/gems/2.2.0/gems/bunny-1.2.1/lib/bunny/session.rb:705:in `clean_up_on_shutdown'
/root/.rbenv/versions/2.2.1/lib/ruby/gems/2.2.0/gems/bunny-1.2.1/lib/bunny/session.rb:698:in `clean_up_and_fail_on_connection_close!'
/root/.rbenv/versions/2.2.1/lib/ruby/gems/2.2.0/gems/bunny-1.2.1/lib/bunny/session.rb:533:in `handle_frame'
/root/.rbenv/versions/2.2.1/lib/ruby/gems/2.2.0/gems/bunny-1.2.1/lib/bunny/reader_loop.rb:81:in `run_once'
/root/.rbenv/versions/2.2.1/lib/ruby/gems/2.2.0/gems/bunny-1.2.1/lib/bunny/reader_loop.rb:33:in `block in run_loop'
/root/.rbenv/versions/2.2.1/lib/ruby/gems/2.2.0/gems/bunny-1.2.1/lib/bunny/reader_loop.rb:30:in `loop'
/root/.rbenv/versions/2.2.1/lib/ruby/gems/2.2.0/gems/bunny-1.2.1/lib/bunny/reader_loop.rb:30:in `run_loop'
E, [2016-11-19T13:02:13.269868 #11969] ERROR -- : Bunny::ShutdownSignal (Bunny::ShutdownSignal)
/root/.rbenv/versions/2.2.1/lib/ruby/gems/2.2.0/gems/bunny-1.2.1/lib/bunny/reader_loop.rb:94:in `raise'
/root/.rbenv/versions/2.2.1/lib/ruby/gems/2.2.0/gems/bunny-1.2.1/lib/bunny/reader_loop.rb:94:in `raise'
/root/.rbenv/versions/2.2.1/lib/ruby/gems/2.2.0/gems/bunny-1.2.1/lib/bunny/session.rb:796:in `maybe_shutdown_reader_loop'
/root/.rbenv/versions/2.2.1/lib/ruby/gems/2.2.0/gems/bunny-1.2.1/lib/bunny/session.rb:705:in `clean_up_on_shutdown'
/root/.rbenv/versions/2.2.1/lib/ruby/gems/2.2.0/gems/bunny-1.2.1/lib/bunny/session.rb:698:in `clean_up_and_fail_on_connection_close!'
/root/.rbenv/versions/2.2.1/lib/ruby/gems/2.2.0/gems/bunny-1.2.1/lib/bunny/session.rb:533:in `handle_frame'
/root/.rbenv/versions/2.2.1/lib/ruby/gems/2.2.0/gems/bunny-1.2.1/lib/bunny/reader_loop.rb:81:in `run_once'
/root/.rbenv/versions/2.2.1/lib/ruby/gems/2.2.0/gems/bunny-1.2.1/lib/bunny/reader_loop.rb:33:in `block in run_loop'
/root/.rbenv/versions/2.2.1/lib/ruby/gems/2.2.0/gems/bunny-1.2.1/lib/bunny/reader_loop.rb:30:in `loop'
/root/.rbenv/versions/2.2.1/lib/ruby/gems/2.2.0/gems/bunny-1.2.1/lib/bunny/reader_loop.rb:30:in `run_loop'
E, [2016-11-19T13:02:13.269914 #11969] ERROR -- : Connection-level error: CONNECTION_FORCED - broker forced connection closure with reason 'shutdown' (Bunny::ConnectionForced)
/root/.rbenv/versions/2.2.1/lib/ruby/gems/2.2.0/gems/bunny-1.2.1/lib/bunny/consumer_work_pool.rb:55:in `join'
/root/.rbenv/versions/2.2.1/lib/ruby/gems/2.2.0/gems/bunny-1.2.1/lib/bunny/consumer_work_pool.rb:55:in `block in join'
/root/.rbenv/versions/2.2.1/lib/ruby/gems/2.2.0/gems/bunny-1.2.1/lib/bunny/consumer_work_pool.rb:55:in `each'
/root/.rbenv/versions/2.2.1/lib/ruby/gems/2.2.0/gems/bunny-1.2.1/lib/bunny/consumer_work_pool.rb:55:in `join'
/root/coinroots/lib/daemons/amqp_daemon.rb:81:in `<top (required)>'
/root/.rbenv/versions/2.2.1/lib/ruby/gems/2.2.0/gems/daemons-1.1.9/lib/daemons/application.rb:203:in `load'
/root/.rbenv/versions/2.2.1/lib/ruby/gems/2.2.0/gems/daemons-1.1.9/lib/daemons/application.rb:203:in `start_load'
/root/.rbenv/versions/2.2.1/lib/ruby/gems/2.2.0/gems/daemons-1.1.9/lib/daemons/application.rb:298:in `start'
/root/.rbenv/versions/2.2.1/lib/ruby/gems/2.2.0/gems/daemons-1.1.9/lib/daemons/controller.rb:70:in `run'
/root/.rbenv/versions/2.2.1/lib/ruby/gems/2.2.0/gems/daemons-1.1.9/lib/daemons.rb:147:in `block in run'
/root/.rbenv/versions/2.2.1/lib/ruby/gems/2.2.0/gems/daemons-1.1.9/lib/daemons/cmdline.rb:109:in `call'
/root/.rbenv/versions/2.2.1/lib/ruby/gems/2.2.0/gems/daemons-1.1.9/lib/daemons/cmdline.rb:109:in `catch_exceptions'
/root/.rbenv/versions/2.2.1/lib/ruby/gems/2.2.0/gems/daemons-1.1.9/lib/daemons.rb:146:in `run'
/root/.rbenv/versions/2.2.1/lib/ruby/gems/2.2.0/gems/daemons-rails-1.2.1/lib/daemons/rails.rb:16:in `run'
lib/daemons/deposit_coin_ctl:11:in `<main>'
E, [2016-11-19T13:02:13.269959 #11969] ERROR -- : no such frame (ArgumentError)
/root/.rbenv/versions/2.2.1/lib/ruby/gems/2.2.0/gems/binding_of_caller-0.7.2/lib/binding_of_caller/mri2.rb:25:in `frame_binding'
/root/.rbenv/versions/2.2.1/lib/ruby/gems/2.2.0/gems/binding_of_caller-0.7.2/lib/binding_of_caller/mri2.rb:25:in `block (2 levels) in callers'
/root/.rbenv/versions/2.2.1/lib/ruby/gems/2.2.0/gems/binding_of_caller-0.7.2/lib/binding_of_caller/mri2.rb:23:in `loop'
/root/.rbenv/versions/2.2.1/lib/ruby/gems/2.2.0/gems/binding_of_caller-0.7.2/lib/binding_of_caller/mri2.rb:23:in `block in callers'
/root/.rbenv/versions/2.2.1/lib/ruby/gems/2.2.0/gems/binding_of_caller-0.7.2/lib/binding_of_caller/mri2.rb:21:in `open'
/root/.rbenv/versions/2.2.1/lib/ruby/gems/2.2.0/gems/binding_of_caller-0.7.2/lib/binding_of_caller/mri2.rb:21:in `callers'
/root/.rbenv/versions/2.2.1/lib/ruby/gems/2.2.0/gems/better_errors-1.1.0/lib/better_errors/core_ext/exception.rb:9:in `block in <class:Exception>'
/root/.rbenv/versions/2.2.1/lib/ruby/gems/2.2.0/gems/bunny-1.2.1/lib/bunny/cruby/socket.rb:41:in `read_nonblock'
/root/.rbenv/versions/2.2.1/lib/ruby/gems/2.2.0/gems/bunny-1.2.1/lib/bunny/cruby/socket.rb:41:in `block in read_fully'
/root/.rbenv/versions/2.2.1/lib/ruby/gems/2.2.0/gems/bunny-1.2.1/lib/bunny/cruby/socket.rb:40:in `loop'
/root/.rbenv/versions/2.2.1/lib/ruby/gems/2.2.0/gems/bunny-1.2.1/lib/bunny/cruby/socket.rb:40:in `read_fully'
/root/.rbenv/versions/2.2.1/lib/ruby/gems/2.2.0/gems/bunny-1.2.1/lib/bunny/transport.rb:199:in `read_next_frame'
/root/.rbenv/versions/2.2.1/lib/ruby/gems/2.2.0/gems/bunny-1.2.1/lib/bunny/reader_loop.rb:63:in `run_once'
/root/.rbenv/versions/2.2.1/lib/ruby/gems/2.2.0/gems/bunny-1.2.1/lib/bunny/reader_loop.rb:33:in `block in run_loop'
/root/.rbenv/versions/2.2.1/lib/ruby/gems/2.2.0/gems/bunny-1.2.1/lib/bunny/reader_loop.rb:30:in `loop'
/root/.rbenv/versions/2.2.1/lib/ruby/gems/2.2.0/gems/bunny-1.2.1/lib/bunny/reader_loop.rb:30:in `run_loop'
E, [2016-11-19T13:02:13.270018 #11969] ERROR -- : Resource temporarily unavailable - read would block (IO::EAGAINWaitReadable)
/root/.rbenv/versions/2.2.1/lib/ruby/gems/2.2.0/gems/bunny-1.2.1/lib/bunny/cruby/socket.rb:41:in `read_nonblock'
/root/.rbenv/versions/2.2.1/lib/ruby/gems/2.2.0/gems/bunny-1.2.1/lib/bunny/cruby/socket.rb:41:in `block in read_fully'
/root/.rbenv/versions/2.2.1/lib/ruby/gems/2.2.0/gems/bunny-1.2.1/lib/bunny/cruby/socket.rb:40:in `loop'
/root/.rbenv/versions/2.2.1/lib/ruby/gems/2.2.0/gems/bunny-1.2.1/lib/bunny/cruby/socket.rb:40:in `read_fully'
/root/.rbenv/versions/2.2.1/lib/ruby/gems/2.2.0/gems/bunny-1.2.1/lib/bunny/transport.rb:199:in `read_next_frame'
/root/.rbenv/versions/2.2.1/lib/ruby/gems/2.2.0/gems/bunny-1.2.1/lib/bunny/reader_loop.rb:63:in `run_once'
/root/.rbenv/versions/2.2.1/lib/ruby/gems/2.2.0/gems/bunny-1.2.1/lib/bunny/reader_loop.rb:33:in `block in run_loop'
/root/.rbenv/versions/2.2.1/lib/ruby/gems/2.2.0/gems/bunny-1.2.1/lib/bunny/reader_loop.rb:30:in `loop'
/root/.rbenv/versions/2.2.1/lib/ruby/gems/2.2.0/gems/bunny-1.2.1/lib/bunny/reader_loop.rb:30:in `run_loop'
E, [2016-11-19T13:02:13.270120 #11969] ERROR -- : no such frame (ArgumentError)
/root/.rbenv/versions/2.2.1/lib/ruby/gems/2.2.0/gems/binding_of_caller-0.7.2/lib/binding_of_caller/mri2.rb:25:in `frame_binding'
/root/.rbenv/versions/2.2.1/lib/ruby/gems/2.2.0/gems/binding_of_caller-0.7.2/lib/binding_of_caller/mri2.rb:25:in `block (2 levels) in callers'
/root/.rbenv/versions/2.2.1/lib/ruby/gems/2.2.0/gems/binding_of_caller-0.7.2/lib/binding_of_caller/mri2.rb:23:in `loop'
/root/.rbenv/versions/2.2.1/lib/ruby/gems/2.2.0/gems/binding_of_caller-0.7.2/lib/binding_of_caller/mri2.rb:23:in `block in callers'
/root/.rbenv/versions/2.2.1/lib/ruby/gems/2.2.0/gems/binding_of_caller-0.7.2/lib/binding_of_caller/mri2.rb:21:in `open'
/root/.rbenv/versions/2.2.1/lib/ruby/gems/2.2.0/gems/binding_of_caller-0.7.2/lib/binding_of_caller/mri2.rb:21:in `callers'
/root/.rbenv/versions/2.2.1/lib/ruby/gems/2.2.0/gems/better_errors-1.1.0/lib/better_errors/core_ext/exception.rb:9:in `block in <class:Exception>'
/root/.rbenv/versions/2.2.1/lib/ruby/gems/2.2.0/gems/bunny-1.2.1/lib/bunny/cruby/socket.rb:41:in `read_nonblock'
/root/.rbenv/versions/2.2.1/lib/ruby/gems/2.2.0/gems/bunny-1.2.1/lib/bunny/cruby/socket.rb:41:in `block in read_fully'
/root/.rbenv/versions/2.2.1/lib/ruby/gems/2.2.0/gems/bunny-1.2.1/lib/bunny/cruby/socket.rb:40:in `loop'
/root/.rbenv/versions/2.2.1/lib/ruby/gems/2.2.0/gems/bunny-1.2.1/lib/bunny/cruby/socket.rb:40:in `read_fully'
/root/.rbenv/versions/2.2.1/lib/ruby/gems/2.2.0/gems/bunny-1.2.1/lib/bunny/transport.rb:199:in `read_next_frame'
/root/.rbenv/versions/2.2.1/lib/ruby/gems/2.2.0/gems/bunny-1.2.1/lib/bunny/reader_loop.rb:63:in `run_once'
/root/.rbenv/versions/2.2.1/lib/ruby/gems/2.2.0/gems/bunny-1.2.1/lib/bunny/reader_loop.rb:33:in `block in run_loop'
/root/.rbenv/versions/2.2.1/lib/ruby/gems/2.2.0/gems/bunny-1.2.1/lib/bunny/reader_loop.rb:30:in `loop'
/root/.rbenv/versions/2.2.1/lib/ruby/gems/2.2.0/gems/bunny-1.2.1/lib/bunny/reader_loop.rb:30:in `run_loop'
E, [2016-11-19T13:02:13.270158 #11969] ERROR -- : Resource temporarily unavailable - read would block (IO::EAGAINWaitReadable)
/root/.rbenv/versions/2.2.1/lib/ruby/gems/2.2.0/gems/bunny-1.2.1/lib/bunny/cruby/socket.rb:41:in `read_nonblock'
/root/.rbenv/versions/2.2.1/lib/ruby/gems/2.2.0/gems/bunny-1.2.1/lib/bunny/cruby/socket.rb:41:in `block in read_fully'
/root/.rbenv/versions/2.2.1/lib/ruby/gems/2.2.0/gems/bunny-1.2.1/lib/bunny/cruby/socket.rb:40:in `loop'
/root/.rbenv/versions/2.2.1/lib/ruby/gems/2.2.0/gems/bunny-1.2.1/lib/bunny/cruby/socket.rb:40:in `read_fully'
/root/.rbenv/versions/2.2.1/lib/ruby/gems/2.2.0/gems/bunny-1.2.1/lib/bunny/transport.rb:199:in `read_next_frame'
/root/.rbenv/versions/2.2.1/lib/ruby/gems/2.2.0/gems/bunny-1.2.1/lib/bunny/reader_loop.rb:63:in `run_once'
/root/.rbenv/versions/2.2.1/lib/ruby/gems/2.2.0/gems/bunny-1.2.1/lib/bunny/reader_loop.rb:33:in `block in run_loop'
/root/.rbenv/versions/2.2.1/lib/ruby/gems/2.2.0/gems/bunny-1.2.1/lib/bunny/reader_loop.rb:30:in `loop'
/root/.rbenv/versions/2.2.1/lib/ruby/gems/2.2.0/gems/bunny-1.2.1/lib/bunny/reader_loop.rb:30:in `run_loop'
E, [2016-11-19T13:02:13.270200 #11969] ERROR -- : no such frame (ArgumentError)
/root/.rbenv/versions/2.2.1/lib/ruby/gems/2.2.0/gems/binding_of_caller-0.7.2/lib/binding_of_caller/mri2.rb:25:in `frame_binding'
/root/.rbenv/versions/2.2.1/lib/ruby/gems/2.2.0/gems/binding_of_caller-0.7.2/lib/binding_of_caller/mri2.rb:25:in `block (2 levels) in callers'
/root/.rbenv/versions/2.2.1/lib/ruby/gems/2.2.0/gems/binding_of_caller-0.7.2/lib/binding_of_caller/mri2.rb:23:in `loop'
/root/.rbenv/versions/2.2.1/lib/ruby/gems/2.2.0/gems/binding_of_caller-0.7.2/lib/binding_of_caller/mri2.rb:23:in `block in callers'
/root/.rbenv/versions/2.2.1/lib/ruby/gems/2.2.0/gems/binding_of_caller-0.7.2/lib/binding_of_caller/mri2.rb:21:in `open'
/root/.rbenv/versions/2.2.1/lib/ruby/gems/2.2.0/gems/binding_of_caller-0.7.2/lib/binding_of_caller/mri2.rb:21:in `callers'
/root/.rbenv/versions/2.2.1/lib/ruby/gems/2.2.0/gems/better_errors-1.1.0/lib/better_errors/core_ext/exception.rb:9:in `block in <class:Exception>'
/root/.rbenv/versions/2.2.1/lib/ruby/gems/2.2.0/gems/bunny-1.2.1/lib/bunny/cruby/socket.rb:41:in `read_nonblock'
/root/.rbenv/versions/2.2.1/lib/ruby/gems/2.2.0/gems/bunny-1.2.1/lib/bunny/cruby/socket.rb:41:in `block in read_fully'
/root/.rbenv/versions/2.2.1/lib/ruby/gems/2.2.0/gems/bunny-1.2.1/lib/bunny/cruby/socket.rb:40:in `loop'
/root/.rbenv/versions/2.2.1/lib/ruby/gems/2.2.0/gems/bunny-1.2.1/lib/bunny/cruby/socket.rb:40:in `read_fully'
/root/.rbenv/versions/2.2.1/lib/ruby/gems/2.2.0/gems/bunny-1.2.1/lib/bunny/transport.rb:199:in `read_next_frame'
/root/.rbenv/versions/2.2.1/lib/ruby/gems/2.2.0/gems/bunny-1.2.1/lib/bunny/reader_loop.rb:63:in `run_once'
/root/.rbenv/versions/2.2.1/lib/ruby/gems/2.2.0/gems/bunny-1.2.1/lib/bunny/reader_loop.rb:33:in `block in run_loop'
/root/.rbenv/versions/2.2.1/lib/ruby/gems/2.2.0/gems/bunny-1.2.1/lib/bunny/reader_loop.rb:30:in `loop'
/root/.rbenv/versions/2.2.1/lib/ruby/gems/2.2.0/gems/bunny-1.2.1/lib/bunny/reader_loop.rb:30:in `run_loop'
E, [2016-11-19T13:02:13.270230 #11969] ERROR -- : Resource temporarily unavailable - read would block (IO::EAGAINWaitReadable)
/root/.rbenv/versions/2.2.1/lib/ruby/gems/2.2.0/gems/bunny-1.2.1/lib/bunny/cruby/socket.rb:41:in `read_nonblock'
/root/.rbenv/versions/2.2.1/lib/ruby/gems/2.2.0/gems/bunny-1.2.1/lib/bunny/cruby/socket.rb:41:in `block in read_fully'
/root/.rbenv/versions/2.2.1/lib/ruby/gems/2.2.0/gems/bunny-1.2.1/lib/bunny/cruby/socket.rb:40:in `loop'
/root/.rbenv/versions/2.2.1/lib/ruby/gems/2.2.0/gems/bunny-1.2.1/lib/bunny/cruby/socket.rb:40:in `read_fully'
/root/.rbenv/versions/2.2.1/lib/ruby/gems/2.2.0/gems/bunny-1.2.1/lib/bunny/transport.rb:199:in `read_next_frame'
/root/.rbenv/versions/2.2.1/lib/ruby/gems/2.2.0/gems/bunny-1.2.1/lib/bunny/reader_loop.rb:63:in `run_once'
/root/.rbenv/versions/2.2.1/lib/ruby/gems/2.2.0/gems/bunny-1.2.1/lib/bunny/reader_loop.rb:33:in `block in run_loop'
/root/.rbenv/versions/2.2.1/lib/ruby/gems/2.2.0/gems/bunny-1.2.1/lib/bunny/reader_loop.rb:30:in `loop'
/root/.rbenv/versions/2.2.1/lib/ruby/gems/2.2.0/gems/bunny-1.2.1/lib/bunny/reader_loop.rb:30:in `run_loop'
E, [2016-11-19T13:02:13.270265 #11969] ERROR -- : no such frame (ArgumentError)
/root/.rbenv/versions/2.2.1/lib/ruby/gems/2.2.0/gems/binding_of_caller-0.7.2/lib/binding_of_caller/mri2.rb:25:in `frame_binding'
/root/.rbenv/versions/2.2.1/lib/ruby/gems/2.2.0/gems/binding_of_caller-0.7.2/lib/binding_of_caller/mri2.rb:25:in `block (2 levels) in callers'
/root/.rbenv/versions/2.2.1/lib/ruby/gems/2.2.0/gems/binding_of_caller-0.7.2/lib/binding_of_caller/mri2.rb:23:in `loop'
/root/.rbenv/versions/2.2.1/lib/ruby/gems/2.2.0/gems/binding_of_caller-0.7.2/lib/binding_of_caller/mri2.rb:23:in `block in callers'
/root/.rbenv/versions/2.2.1/lib/ruby/gems/2.2.0/gems/binding_of_caller-0.7.2/lib/binding_of_caller/mri2.rb:21:in `open'
/root/.rbenv/versions/2.2.1/lib/ruby/gems/2.2.0/gems/binding_of_caller-0.7.2/lib/binding_of_caller/mri2.rb:21:in `callers'
/root/.rbenv/versions/2.2.1/lib/ruby/gems/2.2.0/gems/better_errors-1.1.0/lib/better_errors/core_ext/exception.rb:9:in `block in <class:Exception>'
/root/.rbenv/versions/2.2.1/lib/ruby/gems/2.2.0/gems/bunny-1.2.1/lib/bunny/cruby/socket.rb:41:in `read_nonblock'
/root/.rbenv/versions/2.2.1/lib/ruby/gems/2.2.0/gems/bunny-1.2.1/lib/bunny/cruby/socket.rb:41:in `block in read_fully'
/root/.rbenv/versions/2.2.1/lib/ruby/gems/2.2.0/gems/bunny-1.2.1/lib/bunny/cruby/socket.rb:40:in `loop'
/root/.rbenv/versions/2.2.1/lib/ruby/gems/2.2.0/gems/bunny-1.2.1/lib/bunny/cruby/socket.rb:40:in `read_fully'
/root/.rbenv/versions/2.2.1/lib/ruby/gems/2.2.0/gems/bunny-1.2.1/lib/bunny/transport.rb:199:in `read_next_frame'
/root/.rbenv/versions/2.2.1/lib/ruby/gems/2.2.0/gems/bunny-1.2.1/lib/bunny/reader_loop.rb:63:in `run_once'
/root/.rbenv/versions/2.2.1/lib/ruby/gems/2.2.0/gems/bunny-1.2.1/lib/bunny/reader_loop.rb:33:in `block in run_loop'
/root/.rbenv/versions/2.2.1/lib/ruby/gems/2.2.0/gems/bunny-1.2.1/lib/bunny/reader_loop.rb:30:in `loop'
/root/.rbenv/versions/2.2.1/lib/ruby/gems/2.2.0/gems/bunny-1.2.1/lib/bunny/reader_loop.rb:30:in `run_loop'
E, [2016-11-19T13:02:13.270292 #11969] ERROR -- : Resource temporarily unavailable - read would block (IO::EAGAINWaitReadable)
/root/.rbenv/versions/2.2.1/lib/ruby/gems/2.2.0/gems/bunny-1.2.1/lib/bunny/cruby/socket.rb:41:in `read_nonblock'
/root/.rbenv/versions/2.2.1/lib/ruby/gems/2.2.0/gems/bunny-1.2.1/lib/bunny/cruby/socket.rb:41:in `block in read_fully'
/root/.rbenv/versions/2.2.1/lib/ruby/gems/2.2.0/gems/bunny-1.2.1/lib/bunny/cruby/socket.rb:40:in `loop'
/root/.rbenv/versions/2.2.1/lib/ruby/gems/2.2.0/gems/bunny-1.2.1/lib/bunny/cruby/socket.rb:40:in `read_fully'
/root/.rbenv/versions/2.2.1/lib/ruby/gems/2.2.0/gems/bunny-1.2.1/lib/bunny/transport.rb:199:in `read_next_frame'
/root/.rbenv/versions/2.2.1/lib/ruby/gems/2.2.0/gems/bunny-1.2.1/lib/bunny/reader_loop.rb:63:in `run_once'
/root/.rbenv/versions/2.2.1/lib/ruby/gems/2.2.0/gems/bunny-1.2.1/lib/bunny/reader_loop.rb:33:in `block in run_loop'
/root/.rbenv/versions/2.2.1/lib/ruby/gems/2.2.0/gems/bunny-1.2.1/lib/bunny/reader_loop.rb:30:in `loop'
/root/.rbenv/versions/2.2.1/lib/ruby/gems/2.2.0/gems/bunny-1.2.1/lib/bunny/reader_loop.rb:30:in `run_loop'
E, [2016-11-19T13:02:13.270337 #11969] ERROR -- : no such frame (ArgumentError)
/root/.rbenv/versions/2.2.1/lib/ruby/gems/2.2.0/gems/binding_of_caller-0.7.2/lib/binding_of_caller/mri2.rb:25:in `frame_binding'
/root/.rbenv/versions/2.2.1/lib/ruby/gems/2.2.0/gems/binding_of_caller-0.7.2/lib/binding_of_caller/mri2.rb:25:in `block (2 levels) in callers'
/root/.rbenv/versions/2.2.1/lib/ruby/gems/2.2.0/gems/binding_of_caller-0.7.2/lib/binding_of_caller/mri2.rb:23:in `loop'
/root/.rbenv/versions/2.2.1/lib/ruby/gems/2.2.0/gems/binding_of_caller-0.7.2/lib/binding_of_caller/mri2.rb:23:in `block in callers'
/root/.rbenv/versions/2.2.1/lib/ruby/gems/2.2.0/gems/binding_of_caller-0.7.2/lib/binding_of_caller/mri2.rb:21:in `open'
/root/.rbenv/versions/2.2.1/lib/ruby/gems/2.2.0/gems/binding_of_caller-0.7.2/lib/binding_of_caller/mri2.rb:21:in `callers'
/root/.rbenv/versions/2.2.1/lib/ruby/gems/2.2.0/gems/better_errors-1.1.0/lib/better_errors/core_ext/exception.rb:9:in `block in <class:Exception>'
/root/.rbenv/versions/2.2.1/lib/ruby/gems/2.2.0/gems/bunny-1.2.1/lib/bunny/cruby/socket.rb:41:in `read_nonblock'
/root/.rbenv/versions/2.2.1/lib/ruby/gems/2.2.0/gems/bunny-1.2.1/lib/bunny/cruby/socket.rb:41:in `block in read_fully'
/root/.rbenv/versions/2.2.1/lib/ruby/gems/2.2.0/gems/bunny-1.2.1/lib/bunny/cruby/socket.rb:40:in `loop'
/root/.rbenv/versions/2.2.1/lib/ruby/gems/2.2.0/gems/bunny-1.2.1/lib/bunny/cruby/socket.rb:40:in `read_fully'
/root/.rbenv/versions/2.2.1/lib/ruby/gems/2.2.0/gems/bunny-1.2.1/lib/bunny/transport.rb:199:in `read_next_frame'
/root/.rbenv/versions/2.2.1/lib/ruby/gems/2.2.0/gems/bunny-1.2.1/lib/bunny/reader_loop.rb:63:in `run_once'
/root/.rbenv/versions/2.2.1/lib/ruby/gems/2.2.0/gems/bunny-1.2.1/lib/bunny/reader_loop.rb:33:in `block in run_loop'
/root/.rbenv/versions/2.2.1/lib/ruby/gems/2.2.0/gems/bunny-1.2.1/lib/bunny/reader_loop.rb:30:in `loop'
/root/.rbenv/versions/2.2.1/lib/ruby/gems/2.2.0/gems/bunny-1.2.1/lib/bunny/reader_loop.rb:30:in `run_loop'
E, [2016-11-19T13:02:13.270369 #11969] ERROR -- : Resource temporarily unavailable - read would block (IO::EAGAINWaitReadable)
/root/.rbenv/versions/2.2.1/lib/ruby/gems/2.2.0/gems/bunny-1.2.1/lib/bunny/cruby/socket.rb:41:in `read_nonblock'
/root/.rbenv/versions/2.2.1/lib/ruby/gems/2.2.0/gems/bunny-1.2.1/lib/bunny/cruby/socket.rb:41:in `block in read_fully'
/root/.rbenv/versions/2.2.1/lib/ruby/gems/2.2.0/gems/bunny-1.2.1/lib/bunny/cruby/socket.rb:40:in `loop'
/root/.rbenv/versions/2.2.1/lib/ruby/gems/2.2.0/gems/bunny-1.2.1/lib/bunny/cruby/socket.rb:40:in `read_fully'
/root/.rbenv/versions/2.2.1/lib/ruby/gems/2.2.0/gems/bunny-1.2.1/lib/bunny/transport.rb:199:in `read_next_frame'
/root/.rbenv/versions/2.2.1/lib/ruby/gems/2.2.0/gems/bunny-1.2.1/lib/bunny/reader_loop.rb:63:in `run_once'
/root/.rbenv/versions/2.2.1/lib/ruby/gems/2.2.0/gems/bunny-1.2.1/lib/bunny/reader_loop.rb:33:in `block in run_loop'
/root/.rbenv/versions/2.2.1/lib/ruby/gems/2.2.0/gems/bunny-1.2.1/lib/bunny/reader_loop.rb:30:in `loop'
/root/.rbenv/versions/2.2.1/lib/ruby/gems/2.2.0/gems/bunny-1.2.1/lib/bunny/reader_loop.rb:30:in `run_loop'
E, [2016-11-19T13:02:13.270415 #11969] ERROR -- : no such frame (ArgumentError)
/root/.rbenv/versions/2.2.1/lib/ruby/gems/2.2.0/gems/binding_of_caller-0.7.2/lib/binding_of_caller/mri2.rb:25:in `frame_binding'
/root/.rbenv/versions/2.2.1/lib/ruby/gems/2.2.0/gems/binding_of_caller-0.7.2/lib/binding_of_caller/mri2.rb:25:in `block (2 levels) in callers'
/root/.rbenv/versions/2.2.1/lib/ruby/gems/2.2.0/gems/binding_of_caller-0.7.2/lib/binding_of_caller/mri2.rb:23:in `loop'
/root/.rbenv/versions/2.2.1/lib/ruby/gems/2.2.0/gems/binding_of_caller-0.7.2/lib/binding_of_caller/mri2.rb:23:in `block in callers'
/root/.rbenv/versions/2.2.1/lib/ruby/gems/2.2.0/gems/binding_of_caller-0.7.2/lib/binding_of_caller/mri2.rb:21:in `open'
/root/.rbenv/versions/2.2.1/lib/ruby/gems/2.2.0/gems/binding_of_caller-0.7.2/lib/binding_of_caller/mri2.rb:21:in `callers'
/root/.rbenv/versions/2.2.1/lib/ruby/gems/2.2.0/gems/better_errors-1.1.0/lib/better_errors/core_ext/exception.rb:9:in `block in <class:Exception>'
/root/.rbenv/versions/2.2.1/lib/ruby/gems/2.2.0/gems/bunny-1.2.1/lib/bunny/cruby/socket.rb:41:in `read_nonblock'
/root/.rbenv/versions/2.2.1/lib/ruby/gems/2.2.0/gems/bunny-1.2.1/lib/bunny/cruby/socket.rb:41:in `block in read_fully'
/root/.rbenv/versions/2.2.1/lib/ruby/gems/2.2.0/gems/bunny-1.2.1/lib/bunny/cruby/socket.rb:40:in `loop'
/root/.rbenv/versions/2.2.1/lib/ruby/gems/2.2.0/gems/bunny-1.2.1/lib/bunny/cruby/socket.rb:40:in `read_fully'
/root/.rbenv/versions/2.2.1/lib/ruby/gems/2.2.0/gems/bunny-1.2.1/lib/bunny/transport.rb:199:in `read_next_frame'
/root/.rbenv/versions/2.2.1/lib/ruby/gems/2.2.0/gems/bunny-1.2.1/lib/bunny/reader_loop.rb:63:in `run_once'
/root/.rbenv/versions/2.2.1/lib/ruby/gems/2.2.0/gems/bunny-1.2.1/lib/bunny/reader_loop.rb:33:in `block in run_loop'
/root/.rbenv/versions/2.2.1/lib/ruby/gems/2.2.0/gems/bunny-1.2.1/lib/bunny/reader_loop.rb:30:in `loop'
/root/.rbenv/versions/2.2.1/lib/ruby/gems/2.2.0/gems/bunny-1.2.1/lib/bunny/reader_loop.rb:30:in `run_loop'
E, [2016-11-19T13:02:13.270444 #11969] ERROR -- : Resource temporarily unavailable - read would block (IO::EAGAINWaitReadable)
/root/.rbenv/versions/2.2.1/lib/ruby/gems/2.2.0/gems/bunny-1.2.1/lib/bunny/cruby/socket.rb:41:in `read_nonblock'
/root/.rbenv/versions/2.2.1/lib/ruby/gems/2.2.0/gems/bunny-1.2.1/lib/bunny/cruby/socket.rb:41:in `block in read_fully'
/root/.rbenv/versions/2.2.1/lib/ruby/gems/2.2.0/gems/bunny-1.2.1/lib/bunny/cruby/socket.rb:40:in `loop'
/root/.rbenv/versions/2.2.1/lib/ruby/gems/2.2.0/gems/bunny-1.2.1/lib/bunny/cruby/socket.rb:40:in `read_fully'
/root/.rbenv/versions/2.2.1/lib/ruby/gems/2.2.0/gems/bunny-1.2.1/lib/bunny/transport.rb:199:in `read_next_frame'
/root/.rbenv/versions/2.2.1/lib/ruby/gems/2.2.0/gems/bunny-1.2.1/lib/bunny/reader_loop.rb:63:in `run_once'
/root/.rbenv/versions/2.2.1/lib/ruby/gems/2.2.0/gems/bunny-1.2.1/lib/bunny/reader_loop.rb:33:in `block in run_loop'
/root/.rbenv/versions/2.2.1/lib/ruby/gems/2.2.0/gems/bunny-1.2.1/lib/bunny/reader_loop.rb:30:in `loop'
/root/.rbenv/versions/2.2.1/lib/ruby/gems/2.2.0/gems/bunny-1.2.1/lib/bunny/reader_loop.rb:30:in `run_loop'
E, [2016-11-19T13:02:13.270472 #11969] ERROR -- : no such frame (ArgumentError)
/root/.rbenv/versions/2.2.1/lib/ruby/gems/2.2.0/gems/binding_of_caller-0.7.2/lib/binding_of_caller/mri2.rb:25:in `frame_binding'
/root/.rbenv/versions/2.2.1/lib/ruby/gems/2.2.0/gems/binding_of_caller-0.7.2/lib/binding_of_caller/mri2.rb:25:in `block (2 levels) in callers'
/root/.rbenv/versions/2.2.1/lib/ruby/gems/2.2.0/gems/binding_of_caller-0.7.2/lib/binding_of_caller/mri2.rb:23:in `loop'
/root/.rbenv/versions/2.2.1/lib/ruby/gems/2.2.0/gems/binding_of_caller-0.7.2/lib/binding_of_caller/mri2.rb:23:in `block in callers'
/root/.rbenv/versions/2.2.1/lib/ruby/gems/2.2.0/gems/binding_of_caller-0.7.2/lib/binding_of_caller/mri2.rb:21:in `open'
/root/.rbenv/versions/2.2.1/lib/ruby/gems/2.2.0/gems/binding_of_caller-0.7.2/lib/binding_of_caller/mri2.rb:21:in `callers'
/root/.rbenv/versions/2.2.1/lib/ruby/gems/2.2.0/gems/better_errors-1.1.0/lib/better_errors/core_ext/exception.rb:9:in `block in <class:Exception>'
/root/.rbenv/versions/2.2.1/lib/ruby/gems/2.2.0/gems/bunny-1.2.1/lib/bunny/cruby/socket.rb:41:in `read_nonblock'
/root/.rbenv/versions/2.2.1/lib/ruby/gems/2.2.0/gems/bunny-1.2.1/lib/bunny/cruby/socket.rb:41:in `block in read_fully'
/root/.rbenv/versions/2.2.1/lib/ruby/gems/2.2.0/gems/bunny-1.2.1/lib/bunny/cruby/socket.rb:40:in `loop'
/root/.rbenv/versions/2.2.1/lib/ruby/gems/2.2.0/gems/bunny-1.2.1/lib/bunny/cruby/socket.rb:40:in `read_fully'
/root/.rbenv/versions/2.2.1/lib/ruby/gems/2.2.0/gems/bunny-1.2.1/lib/bunny/transport.rb:199:in `read_next_frame'
/root/.rbenv/versions/2.2.1/lib/ruby/gems/2.2.0/gems/bunny-1.2.1/lib/bunny/reader_loop.rb:63:in `run_once'
/root/.rbenv/versions/2.2.1/lib/ruby/gems/2.2.0/gems/bunny-1.2.1/lib/bunny/reader_loop.rb:33:in `block in run_loop'
/root/.rbenv/versions/2.2.1/lib/ruby/gems/2.2.0/gems/bunny-1.2.1/lib/bunny/reader_loop.rb:30:in `loop'
/root/.rbenv/versions/2.2.1/lib/ruby/gems/2.2.0/gems/bunny-1.2.1/lib/bunny/reader_loop.rb:30:in `run_loop'
E, [2016-11-19T13:02:13.270500 #11969] ERROR -- : Resource temporarily unavailable - read would block (IO::EAGAINWaitReadable)
/root/.rbenv/versions/2.2.1/lib/ruby/gems/2.2.0/gems/bunny-1.2.1/lib/bunny/cruby/socket.rb:41:in `read_nonblock'
/root/.rbenv/versions/2.2.1/lib/ruby/gems/2.2.0/gems/bunny-1.2.1/lib/bunny/cruby/socket.rb:41:in `block in read_fully'
/root/.rbenv/versions/2.2.1/lib/ruby/gems/2.2.0/gems/bunny-1.2.1/lib/bunny/cruby/socket.rb:40:in `loop'
/root/.rbenv/versions/2.2.1/lib/ruby/gems/2.2.0/gems/bunny-1.2.1/lib/bunny/cruby/socket.rb:40:in `read_fully'
/root/.rbenv/versions/2.2.1/lib/ruby/gems/2.2.0/gems/bunny-1.2.1/lib/bunny/transport.rb:199:in `read_next_frame'
/root/.rbenv/versions/2.2.1/lib/ruby/gems/2.2.0/gems/bunny-1.2.1/lib/bunny/reader_loop.rb:63:in `run_once'
/root/.rbenv/versions/2.2.1/lib/ruby/gems/2.2.0/gems/bunny-1.2.1/lib/bunny/reader_loop.rb:33:in `block in run_loop'
/root/.rbenv/versions/2.2.1/lib/ruby/gems/2.2.0/gems/bunny-1.2.1/lib/bunny/reader_loop.rb:30:in `loop'
/root/.rbenv/versions/2.2.1/lib/ruby/gems/2.2.0/gems/bunny-1.2.1/lib/bunny/reader_loop.rb:30:in `run_loop'
E, [2016-11-19T13:02:13.270560 #11969] ERROR -- : no such frame (ArgumentError)
/root/.rbenv/versions/2.2.1/lib/ruby/gems/2.2.0/gems/binding_of_caller-0.7.2/lib/binding_of_caller/mri2.rb:25:in `frame_binding'
/root/.rbenv/versions/2.2.1/lib/ruby/gems/2.2.0/gems/binding_of_caller-0.7.2/lib/binding_of_caller/mri2.rb:25:in `block (2 levels) in callers'
/root/.rbenv/versions/2.2.1/lib/ruby/gems/2.2.0/gems/binding_of_caller-0.7.2/lib/binding_of_caller/mri2.rb:23:in `loop'
/root/.rbenv/versions/2.2.1/lib/ruby/gems/2.2.0/gems/binding_of_caller-0.7.2/lib/binding_of_caller/mri2.rb:23:in `block in callers'
/root/.rbenv/versions/2.2.1/lib/ruby/gems/2.2.0/gems/binding_of_caller-0.7.2/lib/binding_of_caller/mri2.rb:21:in `open'
/root/.rbenv/versions/2.2.1/lib/ruby/gems/2.2.0/gems/binding_of_caller-0.7.2/lib/binding_of_caller/mri2.rb:21:in `callers'
/root/.rbenv/versions/2.2.1/lib/ruby/gems/2.2.0/gems/better_errors-1.1.0/lib/better_errors/core_ext/exception.rb:9:in `block in <class:Exception>'
/root/.rbenv/versions/2.2.1/lib/ruby/gems/2.2.0/gems/bunny-1.2.1/lib/bunny/cruby/socket.rb:41:in `read_nonblock'
/root/.rbenv/versions/2.2.1/lib/ruby/gems/2.2.0/gems/bunny-1.2.1/lib/bunny/cruby/socket.rb:41:in `block in read_fully'
/root/.rbenv/versions/2.2.1/lib/ruby/gems/2.2.0/gems/bunny-1.2.1/lib/bunny/cruby/socket.rb:40:in `loop'
/root/.rbenv/versions/2.2.1/lib/ruby/gems/2.2.0/gems/bunny-1.2.1/lib/bunny/cruby/socket.rb:40:in `read_fully'
/root/.rbenv/versions/2.2.1/lib/ruby/gems/2.2.0/gems/bunny-1.2.1/lib/bunny/transport.rb:199:in `read_next_frame'
/root/.rbenv/versions/2.2.1/lib/ruby/gems/2.2.0/gems/bunny-1.2.1/lib/bunny/session.rb:990:in `open_connection'
/root/.rbenv/versions/2.2.1/lib/ruby/gems/2.2.0/gems/bunny-1.2.1/lib/bunny/session.rb:255:in `start'
/root/coinroots/lib/daemons/amqp_daemon.rb:17:in `<top (required)>'
/root/.rbenv/versions/2.2.1/lib/ruby/gems/2.2.0/gems/daemons-1.1.9/lib/daemons/application.rb:203:in `load'
/root/.rbenv/versions/2.2.1/lib/ruby/gems/2.2.0/gems/daemons-1.1.9/lib/daemons/application.rb:203:in `start_load'
/root/.rbenv/versions/2.2.1/lib/ruby/gems/2.2.0/gems/daemons-1.1.9/lib/daemons/application.rb:298:in `start'
/root/.rbenv/versions/2.2.1/lib/ruby/gems/2.2.0/gems/daemons-1.1.9/lib/daemons/controller.rb:70:in `run'
/root/.rbenv/versions/2.2.1/lib/ruby/gems/2.2.0/gems/daemons-1.1.9/lib/daemons.rb:147:in `block in run'
/root/.rbenv/versions/2.2.1/lib/ruby/gems/2.2.0/gems/daemons-1.1.9/lib/daemons/cmdline.rb:109:in `call'
/root/.rbenv/versions/2.2.1/lib/ruby/gems/2.2.0/gems/daemons-1.1.9/lib/daemons/cmdline.rb:109:in `catch_exceptions'
/root/.rbenv/versions/2.2.1/lib/ruby/gems/2.2.0/gems/daemons-1.1.9/lib/daemons.rb:146:in `run'
/root/.rbenv/versions/2.2.1/lib/ruby/gems/2.2.0/gems/daemons-rails-1.2.1/lib/daemons/rails.rb:16:in `run'
lib/daemons/deposit_coin_ctl:11:in `<main>'
E, [2016-11-19T13:02:13.270608 #11969] ERROR -- : Resource temporarily unavailable - read would block (IO::EAGAINWaitReadable)
/root/.rbenv/versions/2.2.1/lib/ruby/gems/2.2.0/gems/bunny-1.2.1/lib/bunny/cruby/socket.rb:41:in `read_nonblock'
/root/.rbenv/versions/2.2.1/lib/ruby/gems/2.2.0/gems/bunny-1.2.1/lib/bunny/cruby/socket.rb:41:in `block in read_fully'
/root/.rbenv/versions/2.2.1/lib/ruby/gems/2.2.0/gems/bunny-1.2.1/lib/bunny/cruby/socket.rb:40:in `loop'
/root/.rbenv/versions/2.2.1/lib/ruby/gems/2.2.0/gems/bunny-1.2.1/lib/bunny/cruby/socket.rb:40:in `read_fully'
/root/.rbenv/versions/2.2.1/lib/ruby/gems/2.2.0/gems/bunny-1.2.1/lib/bunny/transport.rb:199:in `read_next_frame'
/root/.rbenv/versions/2.2.1/lib/ruby/gems/2.2.0/gems/bunny-1.2.1/lib/bunny/session.rb:990:in `open_connection'
/root/.rbenv/versions/2.2.1/lib/ruby/gems/2.2.0/gems/bunny-1.2.1/lib/bunny/session.rb:255:in `start'
/root/coinroots/lib/daemons/amqp_daemon.rb:17:in `<top (required)>'
/root/.rbenv/versions/2.2.1/lib/ruby/gems/2.2.0/gems/daemons-1.1.9/lib/daemons/application.rb:203:in `load'
/root/.rbenv/versions/2.2.1/lib/ruby/gems/2.2.0/gems/daemons-1.1.9/lib/daemons/application.rb:203:in `start_load'
/root/.rbenv/versions/2.2.1/lib/ruby/gems/2.2.0/gems/daemons-1.1.9/lib/daemons/application.rb:298:in `start'
/root/.rbenv/versions/2.2.1/lib/ruby/gems/2.2.0/gems/daemons-1.1.9/lib/daemons/controller.rb:70:in `run'
/root/.rbenv/versions/2.2.1/lib/ruby/gems/2.2.0/gems/daemons-1.1.9/lib/daemons.rb:147:in `block in run'
/root/.rbenv/versions/2.2.1/lib/ruby/gems/2.2.0/gems/daemons-1.1.9/lib/daemons/cmdline.rb:109:in `call'
/root/.rbenv/versions/2.2.1/lib/ruby/gems/2.2.0/gems/daemons-1.1.9/lib/daemons/cmdline.rb:109:in `catch_exceptions'
/root/.rbenv/versions/2.2.1/lib/ruby/gems/2.2.0/gems/daemons-1.1.9/lib/daemons.rb:146:in `run'
/root/.rbenv/versions/2.2.1/lib/ruby/gems/2.2.0/gems/daemons-rails-1.2.1/lib/daemons/rails.rb:16:in `run'
lib/daemons/deposit_coin_ctl:11:in `<main>'
E, [2016-11-19T13:02:13.270675 #11969] ERROR -- : no such frame (ArgumentError)
/root/.rbenv/versions/2.2.1/lib/ruby/gems/2.2.0/gems/binding_of_caller-0.7.2/lib/binding_of_caller/mri2.rb:25:in `frame_binding'
/root/.rbenv/versions/2.2.1/lib/ruby/gems/2.2.0/gems/binding_of_caller-0.7.2/lib/binding_of_caller/mri2.rb:25:in `block (2 levels) in callers'
/root/.rbenv/versions/2.2.1/lib/ruby/gems/2.2.0/gems/binding_of_caller-0.7.2/lib/binding_of_caller/mri2.rb:23:in `loop'
/root/.rbenv/versions/2.2.1/lib/ruby/gems/2.2.0/gems/binding_of_caller-0.7.2/lib/binding_of_caller/mri2.rb:23:in `block in callers'
/root/.rbenv/versions/2.2.1/lib/ruby/gems/2.2.0/gems/binding_of_caller-0.7.2/lib/binding_of_caller/mri2.rb:21:in `open'
/root/.rbenv/versions/2.2.1/lib/ruby/gems/2.2.0/gems/binding_of_caller-0.7.2/lib/binding_of_caller/mri2.rb:21:in `callers'
/root/.rbenv/versions/2.2.1/lib/ruby/gems/2.2.0/gems/better_errors-1.1.0/lib/better_errors/core_ext/exception.rb:9:in `block in <class:Exception>'
/root/.rbenv/versions/2.2.1/lib/ruby/gems/2.2.0/gems/bunny-1.2.1/lib/bunny/cruby/socket.rb:41:in `read_nonblock'
/root/.rbenv/versions/2.2.1/lib/ruby/gems/2.2.0/gems/bunny-1.2.1/lib/bunny/cruby/socket.rb:41:in `block in read_fully'
/root/.rbenv/versions/2.2.1/lib/ruby/gems/2.2.0/gems/bunny-1.2.1/lib/bunny/cruby/socket.rb:40:in `loop'
/root/.rbenv/versions/2.2.1/lib/ruby/gems/2.2.0/gems/bunny-1.2.1/lib/bunny/cruby/socket.rb:40:in `read_fully'
/root/.rbenv/versions/2.2.1/lib/ruby/gems/2.2.0/gems/bunny-1.2.1/lib/bunny/transport.rb:199:in `read_next_frame'
/root/.rbenv/versions/2.2.1/lib/ruby/gems/2.2.0/gems/bunny-1.2.1/lib/bunny/session.rb:923:in `init_connection'
/root/.rbenv/versions/2.2.1/lib/ruby/gems/2.2.0/gems/bunny-1.2.1/lib/bunny/session.rb:254:in `start'
/root/coinroots/lib/daemons/amqp_daemon.rb:17:in `<top (required)>'
/root/.rbenv/versions/2.2.1/lib/ruby/gems/2.2.0/gems/daemons-1.1.9/lib/daemons/application.rb:203:in `load'
/root/.rbenv/versions/2.2.1/lib/ruby/gems/2.2.0/gems/daemons-1.1.9/lib/daemons/application.rb:203:in `start_load'
/root/.rbenv/versions/2.2.1/lib/ruby/gems/2.2.0/gems/daemons-1.1.9/lib/daemons/application.rb:298:in `start'
/root/.rbenv/versions/2.2.1/lib/ruby/gems/2.2.0/gems/daemons-1.1.9/lib/daemons/controller.rb:70:in `run'
/root/.rbenv/versions/2.2.1/lib/ruby/gems/2.2.0/gems/daemons-1.1.9/lib/daemons.rb:147:in `block in run'
/root/.rbenv/versions/2.2.1/lib/ruby/gems/2.2.0/gems/daemons-1.1.9/lib/daemons/cmdline.rb:109:in `call'
/root/.rbenv/versions/2.2.1/lib/ruby/gems/2.2.0/gems/daemons-1.1.9/lib/daemons/cmdline.rb:109:in `catch_exceptions'
/root/.rbenv/versions/2.2.1/lib/ruby/gems/2.2.0/gems/daemons-1.1.9/lib/daemons.rb:146:in `run'
/root/.rbenv/versions/2.2.1/lib/ruby/gems/2.2.0/gems/daemons-rails-1.2.1/lib/daemons/rails.rb:16:in `run'
lib/daemons/deposit_coin_ctl:11:in `<main>'
E, [2016-11-19T13:02:13.270708 #11969] ERROR -- : Resource temporarily unavailable - read would block (IO::EAGAINWaitReadable)
/root/.rbenv/versions/2.2.1/lib/ruby/gems/2.2.0/gems/bunny-1.2.1/lib/bunny/cruby/socket.rb:41:in `read_nonblock'
/root/.rbenv/versions/2.2.1/lib/ruby/gems/2.2.0/gems/bunny-1.2.1/lib/bunny/cruby/socket.rb:41:in `block in read_fully'
/root/.rbenv/versions/2.2.1/lib/ruby/gems/2.2.0/gems/bunny-1.2.1/lib/bunny/cruby/socket.rb:40:in `loop'
/root/.rbenv/versions/2.2.1/lib/ruby/gems/2.2.0/gems/bunny-1.2.1/lib/bunny/cruby/socket.rb:40:in `read_fully'
/root/.rbenv/versions/2.2.1/lib/ruby/gems/2.2.0/gems/bunny-1.2.1/lib/bunny/transport.rb:199:in `read_next_frame'
/root/.rbenv/versions/2.2.1/lib/ruby/gems/2.2.0/gems/bunny-1.2.1/lib/bunny/session.rb:923:in `init_connection'
/root/.rbenv/versions/2.2.1/lib/ruby/gems/2.2.0/gems/bunny-1.2.1/lib/bunny/session.rb:254:in `start'
/root/coinroots/lib/daemons/amqp_daemon.rb:17:in `<top (required)>'
/root/.rbenv/versions/2.2.1/lib/ruby/gems/2.2.0/gems/daemons-1.1.9/lib/daemons/application.rb:203:in `load'
/root/.rbenv/versions/2.2.1/lib/ruby/gems/2.2.0/gems/daemons-1.1.9/lib/daemons/application.rb:203:in `start_load'
/root/.rbenv/versions/2.2.1/lib/ruby/gems/2.2.0/gems/daemons-1.1.9/lib/daemons/application.rb:298:in `start'
/root/.rbenv/versions/2.2.1/lib/ruby/gems/2.2.0/gems/daemons-1.1.9/lib/daemons/controller.rb:70:in `run'
/root/.rbenv/versions/2.2.1/lib/ruby/gems/2.2.0/gems/daemons-1.1.9/lib/daemons.rb:147:in `block in run'
/root/.rbenv/versions/2.2.1/lib/ruby/gems/2.2.0/gems/daemons-1.1.9/lib/daemons/cmdline.rb:109:in `call'
/root/.rbenv/versions/2.2.1/lib/ruby/gems/2.2.0/gems/daemons-1.1.9/lib/daemons/cmdline.rb:109:in `catch_exceptions'
/root/.rbenv/versions/2.2.1/lib/ruby/gems/2.2.0/gems/daemons-1.1.9/lib/daemons.rb:146:in `run'
/root/.rbenv/versions/2.2.1/lib/ruby/gems/2.2.0/gems/daemons-rails-1.2.1/lib/daemons/rails.rb:16:in `run'
lib/daemons/deposit_coin_ctl:11:in `<main>'
E, [2016-11-19T13:02:13.282296 #11969] ERROR -- : stream closed (IOError)

E, [2016-11-19T13:02:13.282512 #11969] ERROR -- : failed to allocate memory (NoMemoryError)

E, [2016-11-19T13:02:13.282552 #11969] ERROR -- : stack level too deep (SystemStackError)

E, [2016-11-19T13:02:13.282671 #11969] ERROR -- : exception reentered (fatal)

E, [2016-11-19T13:02:13.337217 #11969] ERROR -- : Value will be available once the pipeline executes. (Redis::FutureNotReady)

